packages/mdc-textfield/adapter.js

```diff
@@ -146,6 +146,33 @@ class MDCTextFieldAdapter {
    * @param {number} normalizedX
    */
   setLineRippleTransformOrigin(normalizedX) {}
+
+  /**
+   * Shakes the floating label.
```

Comment:
What happens if there's no label? Ditto for other methods below.

Reply:
Hmm good point...we have a guard in the component, but not in the foundation. I think what I'll do is move the guard into the foundation instead.